molr-mole-core/src/main/java/io/molr/mole/core/support/domain/StubData.java

```diff
@@ -0,0 +1,58 @@
+package io.molr.mole.core.support.domain;
+
+import io.molr.commons.domain.Mission;
+import io.molr.commons.domain.Placeholder;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+final class StubData {
```

Comment:
Isnt this more something like a StubBuilder? If not, what is the difference to a builder? .

Reply:
This thing feels a bit too 'mutable'....